test/OpenTelemetry.Api.Tests/BaggageTests.cs

```diff
@@ -207,7 +209,7 @@ public void CreateBaggageTest()
             ["key2"] = "value2",
             ["KEY2"] = "VALUE2",
             ["KEY3"] = "VALUE3",
-            ["Key3"] = null,
+            ["Key3"] = null!,
```

Comment:
nit: could you add a brief comment here that explains that adding `null` causes "VALUE3" to be removed?

I had to debug this to understand what was happening.

Reply:
Added